content/en/about/supported-releases/index.md

```diff
@@ -51,17 +50,21 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 | Version         | Currently Supported   | Release Date        | End of Life         | Supported Kubernetes Versions | Untested, but may work on Kubernetes Versions |
 |-----------------|-----------------------|---------------------|---------------------|-------------------------------|-----------------------------------------------|
 | master          | No, development only  |                     |                     |                               |                                               |
-| 1.9             | N/A                   | ~Feb 2021(Expected) | ~Aug 2021(Expected) | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
+| 1.9             | Yes                   | February 9, 2021    | ~Aug 2021(Expected) | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
 | 1.8             | Yes                   | November 10, 2020   | ~May 2021(Expected) | 1.16, 1.17, 1.18, 1.19        | 1.15                                          |
-| 1.7             | Yes                   | August 21, 2020     | ~Feb 2021(Expected) | 1.16, 1.17, 1.18              | 1.15                                          |
+| 1.7             | No                    | August 21, 2020     | Feb 25, 2021        | 1.16, 1.17, 1.18              | 1.15                                          |
 | 1.6             | No                    | May 21, 2020        | November 23, 2020   | 1.15, 1.16, 1.17, 1.18        |                                               |
 | 1.5 and earlier | No                    |                     |                     |                               |                                               |
 
 ## Releases without known Common Vulnerabilities and Exposures (CVEs)
 
-| LTS Release                | Patched versions with no known CVEs  |
+{{< warning >}}
+Istio does not guarantee that minor releases that fall outside the support window have all known CVEs patched.
+Please keep up-to-date and use a supported version.
+{{< /warning >}}
+
+| Minor Releases             | Patched versions with no known CVEs  |
 |----------------------------|--------------------------------------|
-| 1.8.x                      | 1.8.1                                |
-| 1.7.x                      | 1.7.3, 1.7.4, 1.7.5, 1.7.6           |
-| 1.6.x                      | 1.6.11, 1.6.12, 1.6.13, 1.6.14       |
-| 1.5 and earlier            | None                                 |
+| 1.9.x                      | 1.9.0+                               |
```

Comment:
1.9.1+ (Envoy released a zero-day today, so we know that 1.9.0 has known CVEs).

Reply:
fixed